README.md

```diff
@@ -126,6 +126,12 @@ To build a Docker image, checkout a specific version of the repository and run:
 
 	docker build -t my:tag -f cmd/shfmt/Dockerfile .
 
+This creates an image that only includes shfmt. Alternatively, if you want an
+image that includes alpine, add `--target alpine`.
```

Comment:
is this still relevant, as there's only one target?

Reply:
Sure it's relevant. Why do you say there is only one target? You can build either without specifying a target, in which case you get the minimal *scratch* image, or you can build with `--target alpine`, in which case you get the alpine image.

Reply:
wasn't aware of the target thing :) and missed the 'scratch' part (as that's a new thing and mdvan said something about distro's and just sticking with alpine)

from your reaction I gather a) we'll have at least 2 images, alpine and 'scratch' (makes perfect sense) and in that case, yes this is relevant :) sorry for the noise.